sddir.go

```diff
@@ -2,34 +2,34 @@ package main
 
 import (
 	"encoding/json"
+	log "github.com/sirupsen/logrus"
```

Comment:
All changes to `sddir.go` were simply `go fmt`.

Reply:
cool. thought I had that running on save, but clearly I need to go spend two hours on my Emacs config.